runtime/gc_base/RootScanner.cpp

```diff
@@ -784,6 +784,24 @@ MM_RootScanner::scanContinuationObjects(MM_EnvironmentBase *env)
 	reportScanningEnded(RootScannerEntity_ContinuationObjects);
 }
 
+bool
+MM_RootScanner::isContinuationListEmpty(MM_EnvironmentBase *env)
+{
+	bool ret = true;
+	MM_ContinuationObjectList *continuationObjectList = _extensions->getContinuationObjectLists();
+	while(ret && (NULL != continuationObjectList)) {
```

Comment:
add space between 'while' and '('
more importantly, for region based GCs, is this iterate through as many sublists as there are regions - we dont remove empty sublists from the main list?

Reply:
A bit of a concern for Balanced GC, for post JDK19, but for those workloads that have very little or no use of VTs - we will iterate through all region's sublists to find that there are no continuation objects to scan. Probably this can be optimized further, but I'm ok for now.